includes/polling/storage/CISCO-FLASH-MIB.inc.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
Could you please rename both files to "cisco.inc.php" instead ?

Reply:
Hi, I did originally named them like this but polling was not working because of https://github.com/librenms/librenms/blob/master/includes/polling/storage.inc.php I think, which is expecting the mib name. I will try again.

Reply:
This old code is a little wonky.  When you call `discover_storage()` the "mib" name is just the file name it will use for polling.  It actually has no other meaning.